backend/scripts/sync_prod_to_sqlite.py

```
@@ -0,0 +1,288 @@
+
+
+def restore(dump_path):
+    log(f'Restoring {dump_path.name} into local Postgres')
+    run(['docker', 'cp', str(dump_path), f'{CONTAINER}:/tmp/dump.sql'])
+    # The volume is fresh and the dump carries --clean --if-exists, so no
+    # statement may fail; abort loudly rather than flow a partial restore
+    # into the SQLite conversion.
+    run([
+        'docker', 'exec', '-e', f'PGPASSWORD={PG_PASSWORD}', CONTAINER,
+        'psql', '-q', '-U', 'postgres', '-d', 'postgres',
+        '-v', 'ON_ERROR_STOP=1', '-f', '/tmp/dump.sql',
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/dump.sql"

(S108)